### ISSUE TITLE
Allow for multiple guards in transitions

### ISSUE THREAD

**Issue by mattpocock** [matthewp/liblucy#28]
```
initial state firstState {
  go => guard(isAllowed) => secondState
}

state secondState {}

state notAllowedState {}
```

Given a state machine like the one above, how would one default to going to the `notAllowedState` if `isAllowed` evaluated to false?

**Comment by matthewp** [comment]
Wouldn't you write this like:

```
initial state firstState {
  go => guard(isAllowed) => secondState
  go => notAllowedState
}

state secondState {}

state notAllowedState {}
```

Or no?

**Comment by mattpocock** [comment]
That was the syntax my brain first reached for - but it didn't seem to work. It appeared to print two 'go' events with different transitions.

**Comment by matthewp** [comment]
Ah ok, bug then.

**Comment by matthewp** [comment]
Fixed in 0.2.2: https://github.com/matthewp/liblucy/releases/tag/v0.2.2
